File: 실버 5 (4문제)/1475_방 번호.c
// 참고 링크 : https://moon-gd.tistory.com/39
#define _CRT_SECURE_NO_WARNINGS
#include <stdio.h>

int main() {
	int number[10] = { 0 };
	int N = 0, max = 0;

	scanf("%d", &N);

	while (N) {
		number[N % 10]++;
		N /= 10;
	}

	if ((number[6] + number[9]) % 2 == 1) {
		number[6] = (number[6] + number[9] + 1) / 2;
	}
	else {
		number[6] = (number[6] + number[9]) / 2;
	}

	for (int i = 0; i < 9; i++) {
		if (max < number[i]) {
			max = number[i];
		}
	}

	printf("%d", max);

	return 0;
}

File: 실버 5 (7문제)/1475_방 번호.py
# 참고 링크 : https://moon-gd.tistory.com/39
N = int(input())
numbers = [0 for _ in range(10)]

while N:
    numbers[N % 10] += 1
    N //= 10

if (numbers[6] + numbers[9]) % 2 == 1:
    numbers[6] = (numbers[6] + numbers[9] + 1) // 2
else:
    numbers[6] = (numbers[6] + numbers[9]) // 2

print(max(numbers[:9]))
